pkg/backends/options.go

```diff
@@ -1,19 +1,20 @@
-package internal
+package backends
 
 import (
-	clientSetV "github.com/hellofresh/kangal/pkg/kubernetes/generated/clientset/versioned"
 	"go.uber.org/zap"
 
 	"k8s.io/client-go/kubernetes"
 	coreListersV1 "k8s.io/client-go/listers/core/v1"
+
+	clientSetV "github.com/hellofresh/kangal/pkg/kubernetes/generated/clientset/versioned"
 )
 
 // Option allows to configure backends
-type Option func(*Registry)
+type Option func(*registry)
```

Comment:
Why not use the interface?

Reply:
all the options here are meant to work only with `registry` and any other implementation